twitter/Models/tweet.h

```diff
@@ -0,0 +1,28 @@
+//
+//  tweet.h
+//  twitter
+//
+//  Created by Nicholas Wayoe on 6/29/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "User.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@interface tweet : NSObject
+@property (nonatomic, strong) NSString *idStr; // For favoriting, retweeting & replying
```

Comment:
I think this might just be copy pasta but avoid using abbreviations like this. `idStr` is not a common acronym or abbreviation. Better to use `id` or `identifier`. We know it's a string.